lib/js/src/manager/LifecycleConfig.js

```diff
@@ -365,6 +367,34 @@ class LifecycleConfig {
                     this.setMinimumRpcVersion(version);
                     break;
                 }
+                case 'appIcon': {
+                    // split the file path by slashes
+                    const fileNameSplit = manifest[key].split(/[\/\\]/g);
```

Comment:
```suggestion
                    const fileNameSplit = manifest[key].split(/[/\\]/g);
```
Removed a `\`. Message from the linter: `Unnecessary escape character: \/`.